packages/support/lib/fs.js

```diff
@@ -48,9 +48,23 @@ const fs = {
     return true;
   },
 
+  /**
+   * Returns `true` if `path` is executable
+   * @param {PathLike} path
+   * @returns {Promise<boolean>}
+   */
+  async isExecutable(path) {
```

Comment:
should this method eventually verify if the given path exists?

Reply:
if it doesn't, `fs.access` will reject

Reply:
I mean if this makes sense for the client code, which is going to call this API. At least we should mention this in the docstrings that `false` is returned for non-existing files. Also, it probably does not make sense on Windows (same as X_OK constant in Node.js

Reply:
OK, I can note that the function will never reject.

re: Windows, I see https://github.com/nodejs/node/issues/19192 which makes me think we just need to check for file existence on that platform.  I'll modify the function accordingly.

Reply:
oh, it's even worse than that.  we will need to actually do an `fs.stat()` on windows since `fs.access()` is seemingly a no-op (and we don't want to use `fs.existsSync` _and_ `fs.exists` is deprecated)